apps/chat/src/store/conversations/conversations.selectors.ts

```diff
@@ -820,3 +820,10 @@ export const selectIsSelectedConversationsWithSchema = createSelector(
   [selectSelectedConversations],
   (conversations) => conversations.some(isConversationWithFormSchema),
 );
+
+export const selectAction = createSelector(
+  [rootSelector],
+  (state: ConversationsState) => {
+    return state.preselectedAction;
+  },
+);
```

Comment:
don't need `createSelector` here
```suggestion
export const selectAction = (state: RootState) =>
  rootSelector(state).preselectedAction;
```

Reply:
Fixed